extensions/pkg/controller/backupentry/mapper.go

```diff
@@ -63,23 +69,27 @@ func (m *secretToBackupEntryMapper) Map(obj client.Object) []reconcile.Request {
 
 // SecretToBackupEntryMapper returns a mapper that returns requests for BackupEntry whose
 // referenced secrets have been modified.
-func SecretToBackupEntryMapper(client client.Client, predicates []predicate.Predicate) extensionshandler.Mapper {
-	return &secretToBackupEntryMapper{client, predicates}
+func SecretToBackupEntryMapper(stopCh <-chan struct{}, client client.Client, predicates []predicate.Predicate) extensionshandler.Mapper {
+	return &secretToBackupEntryMapper{stopCh, client, predicates}
```

Comment:
Can you use `InjectStopChannel` instead of passing this here? I think, there is no other way of retrieving a manager's stop channel.

Something like https://github.com/gardener/gardener/blob/076ee66bdccd77a0ee2b419a2e705b77068f269c/extensions/pkg/controller/worker/mapper.go#L66-L69